ficlatte/settings-example.py

```diff
@@ -1,106 +0,0 @@
-"""
-Django settings for ficlatte project.
```

Comment:
Yeah, I readded the file back with the second commit. I wasn't thinking when I renamed the file instead of copying it and modifying the copy.